mediagrains/comparison/options.py

```diff
@@ -14,39 +14,35 @@
 # limitations under the License.
 #
 
-"""Options can be passed to comparisons which are constructed using the objects Exclude, Include, and
-ExpectedDifference. These three objects provide a convenient (and similar) interface. By accessing attributes
-of these objects that have the same names as attributes of the objects to be compared you can identify which
-attributes to refer to.
+"""Please refer to the README for in-depth information on how to use options in practice.
+The Options make it easy to exclude, include and expect differences between different identifiers.
+Primarily as syntactic sugar the Exclude and ExpectedDifference objects are exported to make it easier to construct
+ComparisonOptions in a simple fashion.
 
-eg.
+This code is difficult to explain in-line, hence, lets demonstrate how an exclude works.
+An example invocation:
 
-options.Include.creation_timestamp
+from mediagrains.comparison.options import Exclude
+compare_grain(a, b, Exclude.creation_timestamp)
```

Comment:
Nit to match the style in the README
```suggestion
>>> from mediagrains.comparison.options import Exclude
>>> compare_grain(a, b, Exclude.creation_timestamp)
```